app/assets/javascripts/discourse/controllers/topic.js.es6

```diff
@@ -679,6 +679,24 @@ export default Controller.extend(bufferedProperty("model"), {
       }
     },
 
+    toggleFeaturedOnProfile() {
+      if (!this.currentUser) return;
+      const toggle = () => {
+        this.model
+          .toggleFeaturedOnProfile(this.currentUser)
+          .catch(popupAjaxError);
+      };
+
+      if (
+        this.currentUser.featured_topic &&
+        this.currentUser.featured_topic.id !== this.model.id
```

Comment:
Could be simplify by:

```
this.get("currentUser.featured_topic.id") !== this.model.id
```

Reply:
These statements are not equivalent, and does not work for what I need. 

I do not want to entire this branch of the if, when the featured_topic is null. `this.get("currentUser.featured_topic.id") !== this.model.id` evaluates to `true` when the `featured_topic` is null.